packages/compiler-cli/src/ngtsc/annotations/component/src/handler.ts

```diff
@@ -278,6 +280,7 @@ export class ComponentDecoratorHandler
     usePoisonedData: boolean;
     enableBlockSyntax: boolean;
     enableLetSyntax: boolean;
+    preserveSignificantWhitespace?: boolean;
```

Comment:
In the compiler we try to avoid optional parameters.
```suggestion
    preserveSignificantWhitespace: boolean;
```